CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - A .cff citation file
 - Phenotype search API endpoint
 - Added pagination to phenotype API
+- Support for connecting to a MongoDB replica set
```

Comment:
This line was in an older branch and in current master is gone. It should be removed since it's replaced by lines 14 and 15